obspy/imaging/waveform.py

```diff
@@ -38,9 +38,6 @@
 import warnings
 
 
-MATPLOTLIB_VERSION = getMatplotlibVersion()
```

Comment:
Not sure actually. Maybe flake8 has some exception for unused all uppercase names?